search/search_service/proxy/es_proxy_v3.py

```diff
@@ -0,0 +1,310 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+from typing import Any, List
+
+from elasticsearch import Elasticsearch
+from elasticsearch_dsl import Q
+from elasticsearch_dsl.query import Match, RankFeature
+
+from search_service.proxy.es_proxy_utils import Resource
+from search_service.proxy.es_proxy_v2 import BOOL_QUERY, ElasticsearchProxyV2
+
+LOGGER = logging.getLogger(__name__)
+
+# ES query constants
+
+DEFAULT_FUZZINESS = "AUTO"
+
+
+class ElasticsearchProxyV3(ElasticsearchProxyV2):
+
+    # map the field name in FE to the field used to filter in ES
+    # note: ES needs keyword field types to filter
+
+    GENERAL_MAPPING = {
+        'key': 'key',
+        'description': 'description',
+        'resource_type': 'resource_type',
+    }
+
+    TABLE_MAPPING = {
+        **GENERAL_MAPPING,
+        'badges': 'badges.keyword',
+        'tag': 'tags.keyword',
```

Comment:
Is this correct as singular "tag"? Noticed that it's the same for dashboard mapping, but it is "tags" for features so just wanted to check

Reply:
yeah this is the way the field was named in frontend for filters for some reason